recipes/libbacktrace/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+
+
+class LibbacktraceConan(ConanFile):
+    name = "libbacktrace"
+    description = "A C library that may be linked into a C/C++ program to produce symbolic backtraces."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ianlancetaylor/libbacktrace"
+    license = "BSD-3-Clause"
```

Comment:
@ianlancetaylor Could you confirm the license for the library, please?

Reply:
Seems the closet https://spdx.org/licenses/BSD-3-Clause.html

Reply:
Yes, this is a 3-clause BSD license.